### DISCUSSION TITLE
Github Copilot VSCode Linux

### DISCUSSION THREAD

**Discussion by hubert-ld** [community/community#124154] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello !

I have searched a the vscode extension on the linux version but I could not find it. Is the extension available ?
I am on Manjaro distribution.

Thanks !

**Comment by MusicMakerOwO** [comment, marked as answer]
I am running arch on this machine and it works just fine, same on my debian server. I don't believe distribution makes a big impact on availability, if at all
https://marketplace.visualstudio.com/items?itemName=GitHub.copilot
